src/Resources/contao/languages/en/tl_user.xlf

```diff
@@ -293,6 +293,27 @@
       <trans-unit id="tl_user.su.1">
         <source>Switch to user ID %s</source>
       </trans-unit>
+      <trans-unit id="tl_user.2fa_legend">
+        <source>Two-factor authentication (2FA)</source>
+      </trans-unit>
+      <trans-unit id="tl_user.use2fa.0">
+        <source>Use two-factor authentication</source>
+      </trans-unit>
+      <trans-unit id="tl_user.use2fa.1">
+        <source>Use two-factor authentication to log into the Contao back end.</source>
+      </trans-unit>
+      <trans-unit id="tl_user.2faQrCode.0">
+        <source>QR code</source>
+      </trans-unit>
+      <trans-unit id="tl_user.2faQrCode.1">
+        <source>Please scan the QR code with a 2FA/TOTP app of your choice (e.g. 1password).</source>
```

Comment:
`1Password` is written with uppercase P. We should also add Google Authenticator, or maybe a link to a tool site?

Reply:
AFAIK it was Google Authenticator, maybe @leofeyer changed?

Reply:
https://github.com/bytehead/core-bundle/commit/b3109397897e918c40aa48b57e9ec4fa15125a07#diff-fecc377bceddaab13c32ad1270a29c99L309

Reply:
Indeed, because Google Authenticator is dead. 😄

Reply:
If Google Authenticator is dead, why do we lower our security standard to what it supports?

Reply:
It's not dead, that's just Leo's opinion 😉

Reply:
🙈